mix.exs

```diff
@@ -64,11 +64,27 @@ defmodule EctoSQL.MixProject do
     ]
   end
 
-  defp ecto_opts do
+  defp ecto_dep do
     if path = System.get_env("ECTO_PATH") do
-      [path: path]
+      {:ecto, path: path}
```

Comment:
previously we were just changing dep opts, so we kept the `~> 3.0.6` version requirement, which breaks when running against ecto master and ecto_sql master:

```
* ecto (../ecto)
  the dependency does not match the requirement "~> 3.0.6", got "3.1.0-dev"
```